web-api/openapi/front_office.yaml

```diff
@@ -108,6 +108,32 @@ paths:
               schema:
                 $ref: '#/components/schemas/UserInteractionError'
 
+  /semantic_search:
+    post:
+      tags:
+        - front office
+      summary: Returns documents similar to the given text.
+      description: |-
+        Returns a list of documents that are semantically similar to the text in the request body.
+        Each document contains the id, the score and the properties.
+        The score is a value between 0 and 1 where a higher value means that the document is more similar to the one in input
+      operationId: createSimilarDocumentsListing
```

Comment:
Yup, but we have an openapi spec validator in the CI, for any POST, then the operationID must start with 'create' and the response code must be 201.
Which is a bit strange IMO, they auto-assume a POST creates something, hence I named it 'createSimilarDocumentsListing' since it well... creates a List

Reply:
ah i see, my bad, thx for clarifying